jobs/rocoto/esnowanl.sh

```diff
@@ -16,7 +14,7 @@ export jobid="${job}.$$"
 ###############################################################
 # setup python path for ioda utilities
 # shellcheck disable=SC2311
-pyiodaPATH="${HOMEgfs}/sorc/gdas.cd/build/lib/python$(detect_py_ver)/"
+pyiodaPATH="${HOMEgfs}/sorc/gdas.cd/build/lib/python${PYTHON_VERSION}/"
 gdasappPATH="${HOMEgfs}/sorc/gdas.cd/sorc/iodaconv/src:${pyiodaPATH}"
 PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}:${gdasappPATH}"
```

Comment:
@CoryMartin-NOAA Would it be appropriate to move these lines to `load_ufsda_modules.sh`?

Reply:
We can do that, they're not needed for every job that sources that script but I don't think it should harm anything

Reply:
Understood.
I think we will need to find a way to handle this in ops because the ecf scripts is where we load the modules.  I have not seen a PYTHONPATH variable setup this way in any of the ops applications.

Reply:
@aerorahul my hope for ops is that we do not have to use any of the python ioda-converters, but rather the script backend directly from the JEDI executable, which should hopefully remove this hack

Reply:
ok. lets do this in the next PR so it gets properly tested